models/ht_neuron.cpp

```diff
@@ -577,7 +624,7 @@ nest::ht_neuron::Buffers_::Buffers_( const Buffers_&, ht_neuron& n )
 nest::ht_neuron::ht_neuron()
   : Archiving_Node()
   , P_()
-  , S_( P_ )
+  , S_( *this, P_ )
```

Comment:
... and same answer, essentially: the state should be initialized to the correct steady-state values.